vue/test/components/configurator/configurator.test.js

```diff
@@ -0,0 +1,11 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Configurator", () => {
+    it("should instantiates the component", () => {
+        const component = base.getComponent("Configurator", {
+            props: { brand: "dummy", model: "cube", version: 52, size: 1000 }
+        });
+        assert.strictEqual(component.emitted("loading").length, 1);

```

Comment:
`loaded` too, right?

Reply:
In this case it is `loading`. It will be difficult to test the `loaded` case since it is difficult to mock the ripe SDK and it is dependent on the async function of binding images.